website/docs/d/event_streams_schema.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "Event Streams"
+layout: "ibm"
+page_title: "IBM: ibm_event_streams_schema"
+description: |-
+  Get information about an IBM Event Streams schema resource.
+---
+
+# ibm_event_streams_schema
+
+Retrieve information about the Event Streams schema data sources. For more information, see documentation on Event Streams [schema registry](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-ES_schema_registry)
```

Comment:
I would say: For more information, about Event Streams, see [schema registry](.....).